simpegEM1D/Fortran/m_rTE_Fortran.f90

```diff
@@ -0,0 +1,503 @@
+module rTE_Fortran
```

Comment:
@leonfoks  Great! This is the core function, and I was thinking about using cython, but you've already transformed this into fortran!

Reply:
So i did the Fortran part because it was easier for me at this time...  If you ever do rewrite in cython it be easier from a maintenance standpoint.  I'd be interested in what the times look like too.

Reply:
@leonfoks I see. It seems both numba and cython are potential options. We can explore this later!